File: core/src/commonMain/kotlin/work/socialhub/kslack/entity/BotIcons.java
package work.socialhub.kslack.entity;

import com.google.gson.annotations.SerializedName;

public class BotIcons {
    @SerializedName("image_36")
    private String image36;
    @SerializedName("image_48")
    private String image48;
    @SerializedName("image_72")
    private String image72;

    public String getImage36() {
        return this.image36;
    }

    public String getImage48() {
        return this.image48;
    }

    public String getImage72() {
        return this.image72;
    }

    public void setImage36(String image36) {
        this.image36 = image36;
    }

    public void setImage48(String image48) {
        this.image48 = image48;
    }

    public void setImage72(String image72) {
        this.image72 = image72;
    }
}

File: core/src/commonMain/kotlin/work/socialhub/kslack/entity/bot/BotIcons.kt
package work.socialhub.kslack.entity.bot

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable
import kotlin.js.JsExport

@JsExport
@Serializable
class BotIcons {

    @SerialName("image_36")
    var image36: String? = null

    @SerialName("image_48")
    var image48: String? = null

    @SerialName("image_72")
    var image72: String? = null
}
